build: dynamically generate mpv.desktop file protocols (#14145)
mpv-player/mpv: DOCS/man/mpv.rst

    ``<number>`` (select playlist with the same index). mpv will list
    the available playlists on loading.

.. warning::

>     Since mpv 0.39.0, the old alias ``bluray://`` was removed and will now
>     instead map to the ffmpeg protocol with the same name if it is available.

Reviewer: Maybe?
```suggestion
    Since mpv 0.39.0, the old alias ``bluray://`` has been removed and will now
    map to the ffmpeg protocol with the same name, if it is available.
```

Author: Changed it to has been but no comma.
